lib/srv/db/server.go

```diff
@@ -169,6 +172,16 @@ func (c *Config) CheckAndSetDefaults(ctx context.Context) (err error) {
 			return trace.Wrap(err)
 		}
 	}
+	if c.Limiter == nil {
+		// Set default limiter if one is not provided.
```

Comment:
Still doesn't answer the question what "default" limiter is. No limits? Some default limits?

Reply:
Updated the comment. By default connection limiting is disabled.